ThrowIT/SceneDelegate.m

```diff
@@ -17,37 +17,33 @@ @implementation SceneDelegate
 
 - (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions {
     PFUser *user = [PFUser currentUser];
-        if (user != nil) {
-                if([user[@"isThrower"] isEqual:@0]){
-                UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-                UITabBarController *timelineTabBarController = [storyboard instantiateViewControllerWithIdentifier:@"TimelineTabBarController"];
-                self.window.rootViewController
-                = timelineTabBarController;
+    if (user != nil) {
+        if([user[@"isThrower"] isEqual:@0]){
```

Comment:
I think there are some places here where we can use the constants instead of the hardcoded strings.